src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const smallTerm = 3;
+  const smallTermDiscount = 20;
+  const bigTerm = 7;
+  const bigTermDiscount = 50;
+  const numberOfDays = days;
+  const basePrice = days * 40;
+
+  if (numberOfDays < smallTerm) {
+    return basePrice;
+  }
+
+  if (numberOfDays < bigTerm) {
+    return basePrice - smallTermDiscount;
+  }
+
+  return basePrice - bigTermDiscount;
```

Comment:
The final return statement applies the big term discount when `numberOfDays` is 7 or more, which is correct. Ensure that this logic is intended as per the task requirements.